components/index.tsx

```diff
@@ -35,3 +45,17 @@ export function NumberText (props: { value: string, decimal?: number, style?: St
     />
   )
 }
+
+export function PrimaryButton (props: React.PropsWithChildren<ButtonProps>): JSX.Element {
+  return (
+    <TouchableOpacity
+      disabled={props.disabled}
+      onPress={props.onPress}
+      style={[tailwind('m-4 mt-8 p-3 rounded flex-row justify-center'), props.disabled === true ? DisabledColorStyle.button : PrimaryColorStyle.button]}
+    >
+      {
+        props.children
+      }
+    </TouchableOpacity>
+  )
+}
```

Comment:
should move this to an individual file

we need to start maintaining them as` components/*.tsx`